docs/mux-logging-service.md

```diff
@@ -49,6 +49,20 @@ There are environment variables for the logging-mux DC config:
 * `FORWARD_LISTEN_HOST` - hostname to listen for secure_forward protocol log
   messages - this is the same as the FQDN in the mux server cert
 
+These are environment variables for the logging-fluentd daemonset config:
+
+* `USE_MUX_CLIENT` - `true`/`false` - default is `false` - if `true`, fluentd
+  will not send logs directly to Elasticsearch, instead it will use
+  `secure_forward` to send logs to mux.
+* `MUX_CLIENT_MODE` - `minimal`/`full_no_k8s_meta` - default is unset - if
+  `minimal`, this is the same behavior as `USE_MUX_CLIENT true`.  If
+  `full_no_k8s_meta`, then fluentd will perform *all* of the data processing
```

Comment:
I really don't like to remove this knob.  I'm afraid it is premature.  That is, we've only begun to do performance testing, and use mux in different scenarios.  The worst case, which is a distinct possibility, is that we find `full_no_k8s_meta` is unsuitable for some cases and we have no easy way to deploy in another mode.
We have a regression test for this mode - it tests both `minimal` and `full_no_k8s_meta` - we have coverage for both deployment scenarios.
I have no problem making `full_no_k8s_meta` the default when deploying OpenShift with mux.